### PR TITLE
fix: ESLint issues in bobuilder-core

### PR DESCRIPTION
**Pull request by JhontSouth**
Addresses # 4684
#minor

## Description
This PR fixes all the new errors and warnings incoming in the latest version of ESLint packages

## Specific Changes
  - Added missing params documentation in some methods.
  - Formatted with required spaces and comment lines required by ESLint,
  - Lint command pointing to main ESLint config file.

## Testing
The following image shows the lint command running without problems in _botbuilder-core_.
image

### CODE REVIEW

**Review by ceciliaavila — CHANGES_REQUESTED** [review]
Please update with base branch's latest changes

**Review by sw-joelmut — CHANGES_REQUESTED** [review]

File: libraries/botbuilder-core/src/botStatePropertyAccessor.ts [hunk 1]
```diff
@@ -46,9 +45,13 @@ export interface StatePropertyAccessor<T = any> {
...
      * ```
      * @param context Context for the current turn of conversation with the user.
-     * @param defaultValue (Optional) default value to copy to the backing storage object if the property isn't found.
      */
     get(context: TurnContext): Promise<T | undefined>;
+
+    /**
+     * @param context Context for the current turn of conversation with the user.
+     * @param defaultValue (Optional) default value to copy to the backing storage object if the property isn't found.
+     */
```
[the remark this answers is not in the document]
Comment (JhontSouth) [reply libraries/botbuilder-core/src/botStatePropertyAccessor.ts]: fixed

File: libraries/botbuilder-core/src/botStatePropertyAccessor.ts [hunk 2]
```diff
@@ -46,9 +45,15 @@ export interface StatePropertyAccessor<T = any> {
      * const value = await myProperty.get(context, { count: 0 });
      * ```
      * @param context Context for the current turn of conversation with the user.
-     * @param defaultValue (Optional) default value to copy to the backing storage object if the property isn't found.
```
[the remark this answers is not in the document]
Comment (JhontSouth) [reply libraries/botbuilder-core/src/botStatePropertyAccessor.ts:49]: the parameter is not in that method

[the remark this answers is not in the document]
Comment (ceciliaavila) [reply libraries/botbuilder-core/src/botStatePropertyAccessor.ts:49]: Oh, it's true, I've got confused with the one below. Thanks!

**Review by ceciliaavila — APPROVED** [review]

**Review by sw-joelmut — APPROVED** [review]

**Comment by JhontSouth** [comment]
Promoted /4827

**Pull request closed without merging**